src/jobs/database_schema_report.yml

```diff
@@ -0,0 +1,75 @@
+---
+description: >
+  Job to generate a schema report for a postgres database using schemaspy. Runs against postgres in docker
+  An HTML report is generated and is available on the artifacts tab of the circleci job
+executor:
+  name: java_localstack_postgres
+  jdk_tag: "21.0"
+  localstack_tag: "3.0"
+  postgres_tag: "16.1"
+parameters:
+  initialise_database:
+    type: string
+    default: "./gradlew check"
```

Comment:
seems a bit naff that the default is to run ktlint and execute _all_ of the tests.  This can take up to 20 minutes on some projects.  Wondering if it is best not to have a default?